github-actions/github-release/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "amp-wp-github-release",
+  "private": true,
+  "version": "1.0.0",
+  "description": "GitHub Action that creates or updates the built zip for a GitHub release.",
+  "author": "XWP, Google, and contributors",
```

Comment:
```suggestion
  "author": "AMP Project Contributors",
```

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3799/commits/de8f578ae5b039d1cc5d28ce00608e147a801a2a.